src/poprox_storage/repositories/experiments.py

```diff
@@ -67,6 +67,22 @@ def store_experiment(
 
         return experiment_id
 
+    def fetch_experiment_by_id(self, experiment_id: str) -> Experiment | None:
+        expt_table = self.tables["experiments"]
+        expt_query = select(expt_table).where(expt_table.c.experiment_id == experiment_id)
+
+        result = self.conn.execute(expt_query).first()
+        if not result:
+            return None
+        else:
+            return Experiment(
+                experiment_id=result.experiment_id,
+                description=result.description,
+                start_date=result.start_date,
+                end_date=result.end_date,
+                phases=[],
```

Comment:
I'm torn on this. My gut says "if we're going to get the experiment we should get all of the experiment", but my other instinct is "if we don't need the extra data we shouldn't fetch it".

Thoughts?

Reply:
I generally agree but didn't want to get hung up on fetching things that aren't used by the export since the export is the linchpin of a bunch of experiment-related work we'd also like to start on. I think there's work to do in the repository classes to make them consistently return domain concept model objects too.

Reply:
Problem for another PR then.